ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { ReactNode, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card, Skeleton, Title } from '@patternfly/react-core';
+import { Chart, ChartAxis, ChartStack, ChartBar, ChartTooltip } from '@patternfly/react-charts';
+
+import { LinkableChartLabel } from 'Components/PatternFly/Charts/LinkableChartLabel';
+import { AlertGroup } from 'services/AlertsService';
+import { severityLabels } from 'messages/common';
+import {
+    navigateOnClickEvent,
+    patternflySeverityTheme,
+    defaultChartHeight as chartHeight,
+    defaultChartBarWidth,
+} from 'utils/chartUtils';
+import { getQueryString } from 'utils/queryStringUtils';
+import { violationsBasePath } from 'routePaths';
+import useResizeObserver from 'hooks/useResizeObserver';
+import useViolationCounts from '../hooks/useViolationCounts';
+import WidgetErrorEmptyState from '../WidgetErrorEmptyState';
+
+type CountsBySeverity = {
+    Low: Record<string, number>;
+    Medium: Record<string, number>;
+    High: Record<string, number>;
+    Critical: Record<string, number>;
+};
+
+function getCountsBySeverity(groups: AlertGroup[]): CountsBySeverity {
+    const result = {
+        Low: {},
+        Medium: {},
+        High: {},
+        Critical: {},
+    };
+
+    groups.forEach(({ group, counts }) => {
+        result.Low[group] = 0;
+        result.Medium[group] = 0;
+        result.High[group] = 0;
+        result.Critical[group] = 0;
+
+        counts.forEach(({ severity, count }) => {
+            result[severityLabels[severity]][group] = parseInt(count, 10);
+        });
+    });
+
+    return result;
+}
+
+function linkForViolationsCategory(category: string) {
+    const queryString = getQueryString({
+        s: { Category: category },
+        sortOption: { field: 'Severity', direction: 'desc' },
+    });
+    return `${violationsBasePath}${queryString}`;
+}
+
+const height = `${chartHeight}px` as const;
+
+function ViolationsByPolicyCategory() {
+    const history = useHistory();
+    const [widgetContainer, setWidgetContainer] = useState<HTMLDivElement | null>(null);
+    const widgetContainerResizeEntry = useResizeObserver(widgetContainer);
+    const { violationCounts, loading, error } = useViolationCounts('CATEGORY', ''); // TODO Implement query filtering
+
+    const countsBySeverity = getCountsBySeverity(violationCounts);
+    const bars = Object.entries(countsBySeverity).map(([severity, counts]) => {
+        const data = Object.entries(counts).map(([group, count]) => ({
+            name: severity,
+            x: group,
+            y: count,
+            label: `${severity}: ${count}`,
```

Comment:
Can you ask for recommendations about accessibility in charts at Developer Office Hours?

In this case, data is conceptually a table. How a chart can provide as close a user experience as possible.